horovod/mxnet/mpi_ops.cc

```diff
@@ -13,21 +13,21 @@
 // limitations under the License.
 // =============================================================================
 
-#include <chrono>
-#include <memory>
-#include <thread>
 #include <atomic>
 
 #include "../common/operations.h"
 #include "adapter.h"
 #include "cuda_util.h"
 #include "mpi_ops.h"
 #include "ready_event.h"
+#include "status_manager.h"
 #include "tensor_util.h"
 
 namespace horovod {
 namespace mxnet {
 
+static StatusManager status_manager;
```

Comment:
Do want want to move the op_count into the status_manager and make it a tuple with the last_error like `(op_count, last_error)`. Would it be helpful for debugging?

Reply:
This is a good one. Maybe we can save the name (which contains the op_count) to make it more informative.